app/risks.py

```diff
@@ -34,10 +36,18 @@
 with open(Path(__file__).parent.joinpath('data', 'departements.geojson'), 'rb') as response:
     departments = json.load(response)
 
+# We fetch the deparment risk score json and store it in the risk_json variable
+risk_json = requests.get(cfg.PYRORISK_FALLBACK).json()
```

Comment:
Could you add the actual live request as well and comment it?
So that the switch will be easy once we solved the inference issues on the pyrorisk side :pray:

Reply:
yes, definitely, I thought I had let it but I forgot apparently!